packages/playground/src/components/custom_toast.vue

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+import { createToast, type ToastOptions } from "mosha-vue-toastify";
+
+const typeToColor: { [key: string]: string } = {
+  success: "#1aa18f",
+  danger: "#FF5252",
+  info: "#1aa18f",
```

Comment:
@ehab-hassan can you check the colors here for the toast and provide colors for the missing ones?

the supported types: `danger`, `default`, `info`, `success`, and `warning`

Reply:
<img width="314" alt="Frame 257" src="https://github.com/threefoldtech/tfgrid-sdk-ts/assets/10484295/58b508c8-6290-406d-baa8-910e7dc013b1">

Default #313131

Success #1AA18F 

Info #2196F3

Warning #FFCC00

Danger #FF5252

- keep the space between the icon and text smaller.
- Make text Capitalise.
- Font 14px or 0.875rem

Reply:
- All applied but i feel that text is too small compared to the whole page. 
![Uploading Screenshot from 2023-10-09 12-44-03.png…]()

Reply:
The font is clear to me and readable no need to make it bigger